src/trans/internal/tpm_pol.F90

```diff
@@ -49,9 +49,9 @@ SUBROUTINE INI_POL(KNSMAX,LDFAST)
                   &*REAL(KKN+KKM-2,JPRD)) )
 DE(KKN,KKM)=SQRT( (REAL(2*KKN+1,JPRD)*REAL(KKN-KKM,JPRD))&
                 &/ (REAL(2*KKN-1,JPRD)*REAL(KKN+KKM,JPRD)) )
-DA(KKN,KKM)=SQRT( (REAL(2*KKN+1,JPRD)*REAL(KKN-KKM,JPRD)&
-                   &*REAL(KKN+KKM,JPRD))&
-                &/  REAL(2*KKN-1,JPRD) )
+!DA(KKN,KKM)=SQRT( (REAL(2*KKN+1,JPRD)*REAL(KKN-KKM,JPRD)&
```

Comment:
Shall we just delete this? You're right, it doesn't seem to be used at all.

Reply:
Do we wait for a second opinion or I should give it a go ?

Reply:
I always nervous about modifying code that is almost as old as I am. Let's see what @marsdeno thinks.

Reply:
It definitely looks pretty safe to me to be deleted!

Reply:
Done